gather-domains.sh

```diff
@@ -82,12 +82,21 @@ sed -i '/[^,]*,[^,]*,Library of Congress,/d;/[^,]*,[^,]*,Government Publishing O
 # We are instead pulling an archived version of the data from GSA/data
 # on GitHub.
 #
-# Note that we have to include .edu, .com, .net, .org, and .us in the
-# --suffix argument because of the domains included in
-# include/current-federal-non-dotgov.csv
-###
+# Note that we have to include .gov as well as.edu, .com, and other
+# top-level domains in the --suffix argument because of the domains
+# included in include/current-federal-non-dotgov.csv.  The full list
+# of these top-level domains is automatically extracted into the
+# TOP_LEVEL_DOMAINS variable.
+###
+TOP_LEVEL_DOMAINS=$(cut --delimiter=, --fields=1 $OUTPUT_DIR/current-federal_modified.csv | \
+                        tail --lines=+2 | \
+                        sed "s/^.*\(\.[^\.]*\)$/\1/" | \
```

Comment:
Thoughts on using `awk` instead of a `sed` regex?

```suggestion
                        awk -F"." '{print $NF;}' | \
```

Reply:
I tend to use `sed` just because I know regex better than I know `awk` syntax.  

1. I also need the leading `.`, so the `awk` code should be `awk -F"." '{print "."$NF;}'`.
2. I can also skip the first line in the same `awk` code:  `awk -F"." '{if (NR > 1) print "."$NF;}'`.  This lets me get rid of the `tail` command.

What do you think of that?

Reply:
Even better! I forgot about the leading `.` so apologies there. I like doing the `tail` work here as well. I tend toward regex as a default too, I just like avoiding it for simpler tasks like this.

Reply:
See commit 2de3935.

Reply:
Thanks for refining my suggestion!